Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Register/UIRegisterViewControllerExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIRegisterViewControllerExtension.swift
```

Comment:
I would give a more generic name to the extension. This extension can be used by other view controllers as well, not only for the registration.